gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -315,12 +327,18 @@
               <p translate class="help-block">
                 Enables sending extended attributes to other devices, and applying incoming extended attributes. May require running with elevated privileges.
               </p>
+              <p class="help-block" ng-if="currentFolder.type == 'sendonly' || currentFolder.type == 'receiveencrypted'" translate translate-value-foldertype="{{friendlyNameFromFolderType(currentFolder.type)}}">
+                Cannot be enabled when the folder type is "{%foldertype%}".
+              </p>
               <label>
                 <input type="checkbox" ng-disabled="currentFolder.type == 'receiveonly' || currentFolder.type == 'receiveencrypted' || currentFolder.syncXattrs" ng-checked="currentFolder.sendXattrs || currentFolder.syncXattrs" ng-model="currentFolder.sendXattrs" /> <span translate>Send Extended Attributes</span>
               </label>
               <p translate class="help-block">
                 Enables sending extended attributes to other devices, but not applying incoming extended attributes. This can have a significant performance impact. Always enabled when "Sync Extended Attributes" is enabled.
               </p>
+              <p class="help-block" ng-if="currentFolder.type == 'receiveonly' || currentFolder.type == 'receiveencrypted'" translate translate-value-foldertype="{{friendlyNameFromFolderType(currentFolder.type)}}">
```

Comment:
We have some convenience function (`has()`? `inArray()`? Or similar) to check the same variable against multiple values.  Might come in handy here, to shorten the code.

Reply:
Changed to `has`.